tests/test_multiple_repositories_integration.py

```diff
@@ -209,11 +209,16 @@ def test_update(self):
     self.assertEqual(sorted(['role1', 'root', 'snapshot', 'targets', 'timestamp']),
         sorted(tuf.roledb.get_rolenames('test_repository2')))
 
+    # Note: refresh() resets the known metadata and updates the latest
+    # top-level metadata.
     self.repository_updater.refresh()
 
-    self.assertEqual(sorted(['role1', 'root', 'snapshot', 'targets', 'timestamp']),
+    self.assertEqual(sorted(['root', 'snapshot', 'targets', 'timestamp']),
         sorted(tuf.roledb.get_rolenames('test_repository1')))
-    self.assertEqual(sorted(['role1', 'root', 'snapshot', 'targets', 'timestamp']),
```

Comment:
I'm confused by this change. How did this result from the PR?

Reply:
refresh() rebuilds the roledb and only refreshes top-level metadata.  See line 212.